realtime_ai_character/static/script.js

```diff
@@ -377,7 +376,6 @@ textButton.addEventListener("click", function() {
     messageButton.click();
     disconnectButton.style.display = "flex";
     textContainer.textContent = "";
-    chatWindow.value += "Hi, my friend, what brings you here today?\n";
```

Comment:
here as well

Reply:
I've changed to send the text from server here:https://github.com/Shaunwei/RealChar/pull/121/files#diff-3a5c2b3a0b3bd08259f644d301f941c71debdb9939d724fb18d06e4fdf499d78R115
So we can delete this line here